bin/run.sh

```diff
@@ -17,5 +17,32 @@ else
   fi
 fi
 
+
+if [[ -f VERSION ]]
+then
+  VERSION=$(cat VERSION)
+else
+  VERSION="Manual Deployment"
+fi
+
+NEW_RELIC_API_KEY=$(
+  echo "${VCAP_SERVICES}" | \
+  jq '.[] | map(select(.name == "tock-credentials-logging")) | .[].credentials.NEW_RELIC_API_KEY' -r
+)
+
+# Append New Relic API key to New Relic INI file for `newrelic-admin` tool
+cat <<EOF >> "${NEW_RELIC_CONFIG_FILE}"
+
+# Adding license key from script $(basename "${0}")
+api_key=${NEW_RELIC_API_KEY}
+EOF
+
+DEPLOYMENT_DESCRIPTION="Recording deployment of ${VERSION}."
+
+echo "${DEPLOYMENT_DESCRIPTION}"
+
+# Record deployment using the New Relic Python Admin CLI
+newrelic-admin record-deploy "${NEW_RELIC_CONFIG_FILE}" "${DEPLOYMENT_DESCRIPTION}"
```

Comment:
Oh, I think I see now... I had asked if this this additional New Relic setup needed or if the initialization in [`wsgi.py`](https://github.com/18F/tock/blob/master/tock/tock/wsgi.py#L16-L25) is enough, but this is being used to log the actual deployment, which is awesome if I'm understanding correctly! :-)